app/models.py

```diff
@@ -45,6 +45,12 @@ class Location(db.Model):
     street_name = db.Column(db.String(100), index=True)
     town_or_city = db.Column(db.String(50), index=True)
     postcode = db.Column(db.String(8), index=True)  # A postcode in the UK can't have more than 8 characters inc. space
+    is_remote = db.Column(db.Boolean, index=True)
+    remote_url = db.Column(db.String, index=True)
+    remote_joining_instructions = db.Column(db.String, index=True)
```

Comment:
Not sure this belongs here

Reply:
If we want locations to be re-useable maybe move this to the actual Masterclass? As it'll be unique data associated with each Masterclass